Detector/DetCommon/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ gaudi_add_library(DetCommon
                  LINK_LIBRARIES GaudiKernel DD4hep ROOT Geant4 DetSegmentation
                  PUBLIC_HEADERS DetCommon)
 
-install(DIRECTORY ${CMAKE_CURRENT_LIST_DIR}/compact DESTINATION Detector/DetCommon)
+install(DIRECTORY ${CMAKE_CURRENT_LIST_DIR}/compact DESTINATION ${CMAKE_INSTALL_DATADIR}/Detector/DetCommon)
```

Comment:
The safest option to avoid possible clashes with other packages would be to use:

```
${CMAKE_INSTALL_DATADIR}/${CMAKE_PROJECT_NAME}/Detector/DetCommon
```

given that `CMAKE_INSTALL_DATADIR` is the same for all packages. Adding the project name we avoid issues with the folders of other projects using the same name.

Reply:
Sure, that's a good suggestion.